skill/response.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2021, Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Determine the response to a time skill request."""
+from datetime import datetime
+
+import pytz
```

Comment:
absolute imports go before relative imports and froms, no?

Reply:
I think there is some confusion here.  The convention should be:

```python
import buiiltin
from builtin import foo

import third_party
from third_party import bar

from mycroft import foobar
from .module_in_same_package import something
```

is that clearer?